docs/check_sphinx.py-tpl

```diff
@@ -1,17 +0,0 @@
-import subprocess
-
-
-def test_linkcheck(tmpdir):
-    doctrees = tmpdir.join("doctrees")
-    htmldir = tmpdir.join("html")
-    subprocess.check_call(
-        ["sphinx-build", "-W", "-blinkcheck", "-d", str(doctrees), ".", str(htmldir)],
-    )
-
-
-def test_build_docs(tmpdir):
-    doctrees = tmpdir.join("doctrees")
-    htmldir = tmpdir.join("html")
-    subprocess.check_call(
-        ["sphinx-build", "-W", "-bhtml", "-d", str(doctrees), ".", str(htmldir)],
-    )
```

Comment:
Will ruff perform these tasks from now on?

Reply:
I replaced them with plain `make` calls that are easier on the eyes!

Reply:
aah in `tox.ini`, looks a lot better indeed :+1: